src/test/java/com/wikia/webdriver/testcases/auth/ForgottenPasswordTests.java

```diff
@@ -69,22 +56,15 @@ public void anonCanRemindPasswordOnUserLoginSpecialPageUsingLowerCaseUserName()
     MailFunctions.deleteAllEmails(credentials.email, credentials.emailPassword);
     WikiBasePageObject base = new WikiBasePageObject();
     base.openWikiPage(wikiURL);
-    SpecialUserLoginPageObject login = base.openSpecialUserLoginOld(wikiURL);
-    SignInPage signIn = new SignInPage(driver);
+    AttachedSignInPage login = base.openSpecialUserLogin(wikiURL);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "login" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)